R/structural.properties.R

```diff
@@ -1220,6 +1224,13 @@ ego <- function(graph, order = 1, nodes = V(graph),
   res
 }
 
+#' @export
+#' @rdname ego
+neighborhood <- function(graph, order = 1, nodes = V(graph),
+                mode = c("all", "out", "in"), mindist = 0) {
+  ego(graph, order = order, nodes = nodes,
+      mode = mode, mindist = mindist)
+}
```

Comment:
This question is so that I can learn good R practices better. Why is this solution preferred to `neighborhood <- ego`?

Reply:
I wanted both versions to appear in the usage section of the docs.

![image](https://github.com/igraph/rigraph/assets/8360597/cdf55323-7830-4ac1-b73c-7106bb83673f)

But let me look whether there are better options.

Reply:
> But let me look whether there are better options.

I just wanted to understand why it's done this way. I was not suggesting a change.